lib/src/tabs/tab-set.component.ts

```diff
@@ -10,52 +10,36 @@ export function throwErrorForMissingRouterLink(tabsWithoutRouterLink: TabCompone
 }
 
 @Component({
-    template: `
-        <div class="hc-{{direction}}-tab-container">
-            <div class="hc-tab-bar-{{direction}}">
-                <div *ngFor="let tab of tabs">
-                    <a *ngIf="routerEnabled" class="hc-tab-{{direction}}"
-                       [routerLink]="tab.routerLink"
-                       routerLinkActive="active"
-                       (click)="setActive(tab)">
-                          {{tab.tabTitle}}
-                    </a>
-                    <a *ngIf="!routerEnabled" class="hc-tab-{{direction}}"
-                       [class.active]="tab.active"
-                       (click)="setActive(tab)">
-                          {{tab.tabTitle}}
-                    </a>
-
-                </div>
-            </div>
-            <div class="hc-tab-content-{{direction}}">
-                 <router-outlet *ngIf="routerEnabled"></router-outlet>
-                 <ng-content *ngIf="!routerEnabled"></ng-content>
-             </div>
-        </div>
-    `,
     selector: `hc-tab-set`,
+    templateUrl: './tab-set.component.html',
     styleUrls: ['../sass/_tabs.scss']
 })
 export class TabSetComponent implements AfterContentInit {
-    @ContentChildren(TabComponent) tabs: QueryList<TabComponent>;
-    @Input() direction: TabDirection = 'vertical';
     routerEnabled: boolean = false;
 
-    constructor(private router: Router, private route: ActivatedRoute) {}
+    @ContentChildren(TabComponent)
```

Comment:
For decorators do we always want them to be above the property they decorate or to the left. I'm fine with either but prefer consistency.

Reply:
I was looking at using a library called [Prettier](https://github.com/prettier/prettier) to handle this stuff so that we have a consistent code base